### DISCUSSION TITLE
Can I add multiple games at once?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#469] [Q&A] [answered]
That would be cool

**Comment by tkashkin** [comment, marked as answer]
You can do it for emulated games. See #475.

**Reply by ghost** [reply]
So does it not work for Linux and Wine Games if I understand that right?

**Reply by tkashkin** [reply]
`.desktop` file import (#423) should be doable.
Other than that I'm not really sure how is this supposed to work.

**Reply by ghost** [reply]
nice
